src/frontend/js/modules/sidebar.js

```diff
@@ -45,14 +50,21 @@ export default class Sidebar {
      * Stores refs to HTML elements needed for correct sidebar work
      */
     this.nodes = {
+      sidebar: null,
       sections: [],
       sidebarContent: null,
       toggler: null,
+      slider: null,
     };
     this.sidebarStorage = new Storage(LOCAL_STORAGE_KEY);
     const storedState = this.sidebarStorage.get();
 
     this.sectionsState = storedState ? JSON.parse(storedState) : {};
+
+    this.sidebarVisibilityStorage = new Storage(SIDEBAR_VISIBILITY_KEY);
+    const storedVisibility = this.sidebarVisibilityStorage.get();
+
+    this.visibility = storedVisibility !== 'false';
```

Comment:
Please, give more clear property name. For example, `isVisible`

Reply:
Fixed